### ISSUE TITLE
[BUG] SteamVR 2.0.8 (non-beta) uses wrong runtime and segfaults

### ISSUE THREAD

**Issue by podiki** [ValveSoftware/SteamVR-for-Linux#629]
**Describe the bug**
The original 2.0 beta didn't work for me (sorry don't have logs handy) but it worked with a later beta. However, with the final 2.0 release it no longer launches but crashes. I'm not sure which version introduced the error, could have been the last beta from a few days ago that I didn't get to try, on Oct. 24th, with its "Fix an issue that prevented SteamVR from starting on Debian/Ubuntu systems". That sounds like the kind of change that leads to this problem.

Here is part of the log, showing the wrong runtime being used (I haven't set one and tried using Scout and then toggling off). A window briefly appears but then it crashes, my system log showing a segfault.

Below you can see a warning about using the Scout runtime for some reason and then nothing useful on the actual crash.

```
/bin/sh\0-c\0/home/john/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=250820 -- /home/john/.local/share/Steam/ubuntu12_32/steam-launch-wrapper -- '/home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh'\0
chdir "/home/john/.local/share/Steam/steamapps/common/SteamVR"

(steam:118): GLib-GIO-CRITICAL **: 17:02:06.972: g_settings_schema_source_lookup: assertion 'source != NULL' failed
vrstartup.sh[3055]: === Fri Oct 27 17:02:07 EDT 2023 ===
vrstartup.sh[3055]: WARNING: launching in legacy LDLP scout, please use sniper SLR
vrstartup.sh[3055]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrenv.sh /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh
vrenv.sh[3055]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh
vrstartup.sh[3055]: WARNING: launching in legacy LDLP scout, please use sniper SLR
vrstartup.sh[3055]: call /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrsetup.sh
vrsetup.sh[3086]: Detected scout LDLP runtime.
vrsetup.sh[3086]: /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher binary has cap_sys_nice privileges
vrstartup.sh[3055]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup-helper.sh
vrstartup-helper.sh[3055]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrstartup
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:02:07 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(3055)
sh: line 1: xdg-mime: command not found
VR Server (v1698182848)

Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:02:07 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(3113)
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 6684 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 9567 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 7356 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 7356 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2349 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2161 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2091 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2017 as invalid for X11 window - use specialized XCB_X11_TO_PID function!

(steam:118): GLib-GIO-CRITICAL **: 17:02:07.229: g_settings_schema_source_lookup: assertion 'source != NULL' failed
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
vrcompositor-launcher.sh[3129]: === Fri Oct 27 17:02:10 EDT 2023 ===
vrcompositor-launcher.sh[3129]: Detected scout LDLP runtime.
vrcompositor-launcher.sh[3129]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher
Using vrcompositor capability proxy
Launching /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:02:10 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(3129)
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:02:20 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(3354)
xkbcommon: ERROR: failed to add default include path /usr/share/X11/xkb
Qt: Failed to create XKB context!
Use QT_XKB_CONFIG_ROOT environmental variable to provide an additional search path, add ':' as separator to provide several search paths and/or make sure that XKB configuration data directory contains recent enough contents, to update please see http://cgit.freedesktop.org/xkeyboard-config/ .
sh: line 1: xdg-icon-resource: command not found
sh: line 1: xdg-mime: command not found
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
CQVRMonitorHeadsetWarning(0x5583ec6e6eb0)  does not have a property named  "green_color"
CQVRMonitorHeadsetWarning(0x5583ec6e6eb0)  does not have a property named  "red_color"
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653

(steam:118): GLib-GIO-CRITICAL **: 17:02:20.696: g_settings_schema_source_lookup: assertion 'source != NULL' failed
vrcompositor-launcher.sh[3374]: === Fri Oct 27 17:02:22 EDT 2023 ===
vrcompositor-launcher.sh[3374]: Detected scout LDLP runtime.
vrcompositor-launcher.sh[3374]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher
Using vrcompositor capability proxy
Launching /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:02:22 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(3374)
Initialize() is failed!!!
./vrwebhelper: error while loading shared libraries: libatk-bridge-2.0.so.0: cannot open shared object file: No such file or directory
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653

(steam:118): GLib-GIO-CRITICAL **: 17:02:33.160: g_settings_schema_source_lookup: assertion 'source != NULL' failed
crash_20231027170233_2.dmp[3608]: Uploading dump (out-of-process)
/tmp/dumps/crash_20231027170233_2.dmp
crash_20231027170233_2.dmp[3608]: Uploading dump (out-of-process)
/tmp/dumps/crash_20231027170233_2.dmp

crash_20231027170233_2.dmp[3608]: Finished uploading minidump (out-of-process): success = yes
crash_20231027170233_2.dmp[3608]: Finished uploading minidump (out-of-process): success = yes

crash_20231027170233_2.dmp[3608]: response: CrashID=bp-c74db2e5-f9de-41c6-b0c1-a26042231027
crash_20231027170233_2.dmp[3608]: response: CrashID=bp-c74db2e5-f9de-41c6-b0c1-a26042231027

crash_20231027170233_2.dmp[3608]: file ''/tmp/dumps/crash_20231027170233_2.dmp'', upload yes: ''CrashID=bp-c74db2e5-f9de-41c6-b0c1-a26042231027''
crash_20231027170233_2.dmp[3608]: file ''/tmp/dumps/crash_20231027170233_2.dmp'', upload yes: ''CrashID=bp-c74db2e5-f9de-41c6-b0c1-a26042231027''

Game 250820 created interface STEAMAPPLIST_INTERFACE_VERSION001 / AppList
Game 250820 created interface STEAMAPPS_INTERFACE_VERSION008 / Apps
Game 250820 created interface STEAMHTMLSURFACE_INTERFACE_VERSION_004 / HTMLSurface
Game 250820 created interface STEAMHTTP_INTERFACE_VERSION002 / HTTP
Game 250820 created interface STEAMINVENTORY_INTERFACE_V002 / Inventory
Game 250820 created interface STEAMMUSICREMOTE_INTERFACE_VERSION001 / MusicRemote
Game 250820 created interface STEAMMUSIC_INTERFACE_VERSION001 / Music
Game 250820 created interface STEAMPARENTALSETTINGS_INTERFACE_VERSION001 / ParentalSettings
Game 250820 created interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 / RemoteStorage
Game 250820 created interface STEAMSCREENSHOTS_INTERFACE_VERSION003 / Screenshots
Game 250820 created interface STEAMUGC_INTERFACE_VERSION010 / UGC
Game 250820 created interface STEAMUSERSTATS_INTERFACE_VERSION011 / UserStats
Game 250820 created interface STEAMVIDEO_INTERFACE_V002 / Video
Game 250820 created interface SteamController006 / Controller
Game 250820 created interface SteamFriends015 / Friends
Game 250820 created interface SteamMatchMaking009 / Matchmaking
Game 250820 created interface SteamMatchMakingServers002 / MatchmakingServers
Game 250820 created interface SteamNetworking005 / Networking
Game 250820 created interface SteamUser019 / User
Game 250820 created interface SteamUtils009 / Utils
Game 250820 method call count for IClientUtils::RecordSteamInterfaceCreation : 22
Game 250820 method call count for IClientUtils::GetAppID : 24
Game 250820 method call count for IClientUser::GetSteamID : 2
Uploaded AppInterfaceStats to Steam
```

and

```
Oct 27 17:02:33 localhost vmunix: [85768.573855] vrmonitor[12923]: segfault at 7f19c691ad90 ip 00007f19c691ad90 sp 00007fff2e64acc8 error 7 in libQt5Gui.so.5[7f19c6600000+5f1000] likely on CPU 11 (core 5, socket 0)
```

**System Information (please complete the following information):**
- Distribution: GNU Guix (Steam runs in a container with FHS structure and all needed libraries, etc.; works great for everything else and SteamVR in the past, up to the many bugs everyone reports on other distros as well)
- SteamVR version: 2.0.8
- Steam client version: latest stable; tried with beta as well
- Opted into Steam client beta?: No (tried with beta as well)
- Graphics driver version:  Mesa 23.2.1
- Gist for SteamVR System Information: can't because it crashes...

**Comment by podiki** [comment]
Note, there is an error about a missing library, even adding that (to the container Steam runs in) has the same outcome and segfault.

```
/bin/sh\0-c\0/home/john/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=250820 -- /home/john/.local/share/Steam/ubuntu12_32/steam-launch-wrapper -- '/home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh'\0
chdir "/home/john/.local/share/Steam/steamapps/common/SteamVR"

(steam:118): GLib-GIO-CRITICAL **: 17:13:03.779: g_settings_schema_source_lookup: assertion 'source != NULL' failed
vrstartup.sh[2126]: === Fri Oct 27 17:13:03 EDT 2023 ===
vrstartup.sh[2126]: WARNING: launching in legacy LDLP scout, please use sniper SLR
vrstartup.sh[2126]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrenv.sh /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh
vrenv.sh[2126]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh
vrstartup.sh[2126]: WARNING: launching in legacy LDLP scout, please use sniper SLR
vrstartup.sh[2126]: call /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrsetup.sh
vrsetup.sh[2157]: Detected scout LDLP runtime.
vrsetup.sh[2157]: /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher binary has cap_sys_nice privileges
vrstartup.sh[2126]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup-helper.sh
vrstartup-helper.sh[2126]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrstartup
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:03 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2126)
sh: line 1: xdg-mime: command not found
VR Server (v1698182848)

Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:03 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2184)
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 6684 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 9567 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 7356 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 7356 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2234 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2349 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2161 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2091 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 2017 as invalid for X11 window - use specialized XCB_X11_TO_PID function!

(steam:118): GLib-GIO-CRITICAL **: 17:13:03.985: g_settings_schema_source_lookup: assertion 'source != NULL' failed
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
vrcompositor-launcher.sh[2200]: === Fri Oct 27 17:13:06 EDT 2023 ===
vrcompositor-launcher.sh[2200]: Detected scout LDLP runtime.
vrcompositor-launcher.sh[2200]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher
Using vrcompositor capability proxy
Launching /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:06 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2200)
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:17 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2425)
xkbcommon: ERROR: failed to add default include path /usr/share/X11/xkb
Qt: Failed to create XKB context!
Use QT_XKB_CONFIG_ROOT environmental variable to provide an additional search path, add ':' as separator to provide several search paths and/or make sure that XKB configuration data directory contains recent enough contents, to update please see http://cgit.freedesktop.org/xkeyboard-config/ .
sh: line 1: xdg-icon-resource: command not found
sh: line 1: xdg-mime: command not found
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
CQVRMonitorHeadsetWarning(0x55baf338c340)  does not have a property named  "green_color"
CQVRMonitorHeadsetWarning(0x55baf338c340)  does not have a property named  "red_color"
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653

(steam:118): GLib-GIO-CRITICAL **: 17:13:17.400: g_settings_schema_source_lookup: assertion 'source != NULL' failed
vrcompositor-launcher.sh[2445]: === Fri Oct 27 17:13:19 EDT 2023 ===
vrcompositor-launcher.sh[2445]: Detected scout LDLP runtime.
vrcompositor-launcher.sh[2445]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher
Using vrcompositor capability proxy
Launching /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:19 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2445)
Initialize() is failed!!!
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561197972149653 [API loaded yes]
Steam_SetMinidumpSteamID:  Setting Steam ID:  76561197972149653
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:29 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2670)

(steam:118): GLib-GIO-CRITICAL **: 17:13:29.916: g_settings_schema_source_lookup: assertion 'source != NULL' failed
crash_20231027171329_2.dmp[2680]: Uploading dump (out-of-process)
/tmp/dumps/crash_20231027171329_2.dmp
crash_20231027171329_2.dmp[2680]: Uploading dump (out-of-process)
/tmp/dumps/crash_20231027171329_2.dmp

crash_20231027171329_2.dmp[2680]: Finished uploading minidump (out-of-process): success = yes
crash_20231027171329_2.dmp[2680]: Finished uploading minidump (out-of-process): success = yes

crash_20231027171329_2.dmp[2680]: response: CrashID=bp-324dc730-5a97-4571-bd37-e77c42231027
crash_20231027171329_2.dmp[2680]: response: CrashID=bp-324dc730-5a97-4571-bd37-e77c42231027

crash_20231027171329_2.dmp[2680]: file ''/tmp/dumps/crash_20231027171329_2.dmp'', upload yes: ''CrashID=bp-324dc730-5a97-4571-bd37-e77c42231027''
crash_20231027171329_2.dmp[2680]: file ''/tmp/dumps/crash_20231027171329_2.dmp'', upload yes: ''CrashID=bp-324dc730-5a97-4571-bd37-e77c42231027''

vrcompositor-launcher.sh[2685]: === Fri Oct 27 17:13:31 EDT 2023 ===
vrcompositor-launcher.sh[2685]: Detected scout LDLP runtime.
vrcompositor-launcher.sh[2685]: exec /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor-launcher
Using vrcompositor capability proxy
Launching /home/john/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
Using breakpad crash handler
Setting breakpad minidump AppID = 250820
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
10/27 17:13:31 Init: Installing breakpad exception handler for appid(250820)/version(2.0.8)/tid(2685)
Game 250820 created interface STEAMAPPLIST_INTERFACE_VERSION001 / AppList
Game 250820 created interface STEAMAPPS_INTERFACE_VERSION008 / Apps
Game 250820 created interface STEAMHTMLSURFACE_INTERFACE_VERSION_004 / HTMLSurface
Game 250820 created interface STEAMHTTP_INTERFACE_VERSION002 / HTTP
Game 250820 created interface STEAMINVENTORY_INTERFACE_V002 / Inventory
Game 250820 created interface STEAMMUSICREMOTE_INTERFACE_VERSION001 / MusicRemote
Game 250820 created interface STEAMMUSIC_INTERFACE_VERSION001 / Music
Game 250820 created interface STEAMPARENTALSETTINGS_INTERFACE_VERSION001 / ParentalSettings
Game 250820 created interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 / RemoteStorage
Game 250820 created interface STEAMSCREENSHOTS_INTERFACE_VERSION003 / Screenshots
Game 250820 created interface STEAMUGC_INTERFACE_VERSION010 / UGC
Game 250820 created interface STEAMUSERSTATS_INTERFACE_VERSION011 / UserStats
Game 250820 created interface STEAMVIDEO_INTERFACE_V002 / Video
Game 250820 created interface SteamController006 / Controller
Game 250820 created interface SteamFriends015 / Friends
Game 250820 created interface SteamMatchMaking009 / Matchmaking
Game 250820 created interface SteamMatchMakingServers002 / MatchmakingServers
Game 250820 created interface SteamNetworking005 / Networking
Game 250820 created interface SteamUser019 / User
Game 250820 created interface SteamUtils009 / Utils
Game 250820 method call count for IClientUtils::RecordSteamInterfaceCreation : 22
Game 250820 method call count for IClientUtils::GetAppID : 24
Game 250820 method call count for IClientUser::GetSteamID : 2
Uploaded AppInterfaceStats to Steam
```

**Comment by podiki** [comment]
I tried the older branches of steamvr that are available and also segfaulted on `libQt5Gui.so.5` so maybe this isn't a steamvr change but from the steam client/runtimes?

**Comment by podiki** [comment]
It appears some files got corrupted and fixed after running the verify files option (despite switching various versions, which I would think would have the same effect). I had narrowed down that the `libQt5.gui.so.5` must be from steamvr (nowhere else on the system), saw it was last modified when I noticed this problem, and did the verify files. The warning about the wrong runtime also went away. So something happened during an update or other random occurrence? Anyway, back to the usual working/bugs of steamvr!
